Lib/pathlib/_abc.py

```diff
@@ -640,23 +640,22 @@ def write_text(self, data, encoding=None, errors=None, newline=None):
         with self.open(mode='w', encoding=encoding, errors=errors, newline=newline) as f:
             return f.write(data)
 
-    def scandir(self):
+    def _scandir(self):
```

Comment:
Do we need this here at all? Can't it just be a helper on the concrete class? Or they can just delegate directly to `os.scandir` (it's all internal details now, so doesn't really matter where it goes)

Reply:
Could you provide more detail? I don't think I follow your suggestion, sorry.

Reply:
It follows on from suggesting that an efficient `walk` probably needs to be on `Path` and not `PathBase` (at which point, `_scandir` way as well also be on `Path`). I'm not sure what order you see the comments in from GitHub.